bartekpacia/mentorship-client
# feat: add confirm password field and show passwords feature to change password popUp

### Description
Add confirm password field with validation and toggle password visibility button on the change password popUp

Fixes #76 

### Flutter Channel:
- [ ] I have used the Flutter Beta channel on my local machine

### Type of Change:

- Code

**Code/Quality Assurance Only**

- New feature (non-breaking change which adds functionality pre-approved by mentors)



### How Has This Been Tested?
Physical Device
Steps:
- Go to the Settings page by clicking on the icon at the top right corner of the homepage.
- Click on the Change Password button to see whether the feature works properly or not
![flutter_01](https://user-images.githubusercontent.com/58745044/83939354-dfb38180-a7f9-11ea-95e5-15590673be76.png)
 

### Checklist:
**Delete irrelevant options.**

- [x] I have commented on my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
